src/Common/Core/Impl/Services/IServiceContainer.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Common.Core.Services {
+    public interface IServiceContainer {
+        /// <summary>
+        /// Provides access to global application services
+        /// </summary>
+        T GetService<T>(Type type = null) where T : class;
```

Comment:
Got it

---
In reply to: [106052444](https://github.com/Microsoft/RTVS/pull/3302#discussion_r106052444) [](ancestors = 106052444)